### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by ravi0dubey**
Created Image Extraction

### CODE REVIEW

File: data_preprocessing/rd_video_image_extractor.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import argparse
+import multiprocessing
+from pathlib import Path
+import cv2
+
```
Comment (Mohit-Ak) [inline data_preprocessing/rd_video_image_extractor.py:19]: Can you please add company header and a comment in PEP standard to describe the function of the file

File: data_preprocessing/rd_video_image_extractor.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    with multiprocessing.Pool() as pool:
+        pool.starmap(process_video, [(video, output_path, frame_rate) for video in video_files])
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("root_dir", type=str)
+    parser.add_argument("output_dir", type=str)
+    parser.add_argument("--frame_rate", type=int, default=1)
+    args = parser.parse_args()
+    main(args.root_dir, args.output_dir, args.frame_rate)
```
Comment (Mohit-Ak) [inline data_preprocessing/rd_video_image_extractor.py:94]: A new line needs to be added at the end of python files by the PEP standards.

**Review by Mohit-Ak — CHANGES_REQUESTED** [review]

File: data_preprocessing/rd_video_image_extractor.py (hunk 2 above)
Comment (ravi0dubey) [reply data_preprocessing/rd_video_image_extractor.py:94]: New Line added at the end . commit number [ce57cd7]

**Review by Mohit-Ak — APPROVED** [review]
Looks good. Please merge it.

**Pull request merged**